cluster/addons/fluentd-elasticsearch/fluentd-es-image/td-agent.conf

```diff
@@ -40,7 +40,8 @@
   path /var/lib/docker/containers/*/*-json.log
   pos_file /var/lib/docker/containers/containers.log.pos
   time_format %Y-%m-%dT%H:%M:%S
-  tag docker.container.*
+  tag docker.*
```

Comment:
Ah, good point. I thought I would get rid of the `containers` component because it was not adding anything. Will fix match block.